README.rst

```diff
@@ -229,14 +229,15 @@ Documentation for other versions
 * `v0.1.0 <https://www.pygmt.org/v0.1.0>`__
 * `v0.0.1a0 <https://www.pygmt.org/0.0.1a0>`__
 
-Compatibility with Python and GMT versions
-------------------------------------------
+Compatibility with GMT and Python/NumPy versions
+------------------------------------------------
 
-======= ========== =========
-PyGMT   GMT        Python
-======= ========== =========
+======= ========== ========= =========
+PyGMT   GMT        Python    NumPy
+======= ========== ========= =========
+0.4.x   >=6.2.0    >=3.7     >=1.17.0
```

Comment:
Is it a good idea to add this here now, or should we do it during the v0.4.0 release process instead?

Reply:
We _should_ add it when we release v0.4.0, but I'm afraid we may forget it during the release process. I think it's fine to add this here.